src/pages/admin/Admin.tsx

```diff
@@ -69,6 +69,7 @@ export const Admin = new Elysia({
           blackPlayerTwo: body.black2Id,
           result: body.match_winner,
           scoreDiff: Number(body.point_difference),
+          createdAt: new Date(`${body.date_played}T${body.time_played}`),
```

Comment:
The backend will expect the date to be in UTC (at least in production), but the DateTimePicker converts it to Europe/Copenhagen before sending to the frontend and there doesn't seem to be any logic for converting it back to UTC again 👀

Reply:
fixed in #49